src/silx/io/specfile/meson.build

```
@@ -0,0 +1,44 @@
+# Definitions of macros specific to specfile
+macros = []
+if target_machine.system() == 'windows'
+    macros = [ '-DSPECFILE_POSIX', '-DWIN32']
+elif target_machine.system() in ['linux', 'darwin', 'dragonfly', 'freebsd', 'gnu']
+    if os.is_file('SPECFILE_USE_GNU_SOURCE')
+        specfile_use_gnu = os.read('SPECFILE_USE_GNU_SOURCE').strip()
+    else
+        res = run_command(py, '-c', 'import os; print(os.environ["SPECFILE_USE_GNU_SOURCE"])',
+                          check:false)
+        if res.returncode() == 0
+            specfile_use_gnu = res.stdout().strip()
+        else
+            specfile_use_gnu = ''
+        endif
+    endif
```

Same, we were not providing file-based config, so I would not add it:

```suggestion
    res = run_command(py, '-c', 'import os; print(os.environ["SPECFILE_USE_GNU_SOURCE"])',
                      check:false)
    if res.returncode() == 0
        specfile_use_gnu = res.stdout().strip()
    else
        specfile_use_gnu = ''
    endif
    endif
```